swri_transform_util/CMakeLists.txt

```diff
@@ -20,6 +20,17 @@ find_package(catkin REQUIRED COMPONENTS
 find_package(OpenCV  REQUIRED core highgui imgproc video)
 find_package(Boost REQUIRED)
 
+if($ENV{ROS_DISTRO} STRLESS "kinetic")
```

Comment:
I think it would be better here to switch on the actual failure (pkg-config), rather than the ROS version. It would be more robust and less dependent on the ROS version correlating wtih the OS. Something like:

```cmake
find_package(PkgConfig REQUIRED)
pkg_check_modules(PROJ QUIET proj)
if (NOT ${PROJ_FOUND})
  set(PROJ_INCLUDE_DIRS /usr/include)
  set(PROJ_LIBRARY_DIRS /usr/lib)
  find_library(PROJ_LIBRARIES proj
    HINTS ${PROJ_LIBRARY_DIRS}
  )
endif()
```

Reply:
Good point.  Changed.